lib/remote_retro/emails.ex

```diff
@@ -0,0 +1,31 @@
+defmodule RemoteRetro.Emails do
+  import Bamboo.Email, except: [from: 2]
+  import Ecto.Query
+  alias RemoteRetro.Repo
+  def action_items_email(email_address, retro_id) do
+    new_email(
+      to: email_address,
+      from: "do-not-reply@remote_retro.dev",
+      subject: "Action items from Retro",
+      text_body: text_retro_action_items(retro_id),
+      html_body: html_retro_action_items(retro_id)
+    )
+  end
+
+  defp text_retro_action_items(retro_id) do
+    retro_action_items(retro_id)
+    |> Enum.join("\n")
+  end
+
+  defp html_retro_action_items(retro_id) do
```

Comment:
Like I said, I don't actually think this is write/html for emails is different than web, and br's aren't really going to cut it, but presumably this basic approach will work with some tweaks.